lighthouse-cli/bin.js

```diff
@@ -15,8 +15,9 @@ const runLighthouse = require('./run').runLighthouse;
 
 const log = require('lighthouse-logger');
 // @ts-ignore
-const perfOnlyConfig = require('../lighthouse-core/config/perf.json');
-const mixedContentConfig = require('../lighthouse-core/config/mixed-content.js');
+const perfOnlyConfig = require('../lighthouse-core/config/perf-config.js');
```

Comment:
does this still need to be `@ts-ignore`? I think that was because it always has an error on trying to require a json file, so it shouldn't be an issue with a js file. OTOH it might be `any` or hopelessly typed :)

Reply:
nope, totally unnecessary now good call 👍